kubernetes/kserve/kserve_wrapper/TorchserveModel.py

```diff
@@ -53,72 +55,103 @@ def __init__(self, name, inference_address, management_address, model_dir):
         self.explainer_host = self.predictor_host
         logging.info("kfmodel Explain URL set to %s", self.explainer_host)
 
-    async def predict(self, request: Dict) -> Dict:
+    async def predict(self, payload: Union[Dict, InferRequest], headers: Dict) -> Dict:
```

Comment:
So,  how does like this look like when you make a request? 
Can you please update the Readme with the new predict call